<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_shadow">
      <property name="styleSheet">
       <string notr="true">QFrame {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(62, 101, 190, 255), stop:1 rgba(81, 47, 154, 255));
border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>18</number>
       </property>
       <property name="topMargin">
        <number>18</number>
       </property>
       <property name="rightMargin">
        <number>18</number>
       </property>
       <property name="bottomMargin">
        <number>18</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(255, 255, 255);
font-size: 16pt;
font-family: &quot;Segoe UI Light&quot;;
font-weight: bold;</string>
         </property>
         <property name="text">
          <string>Parameter</string>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(0, 0, 0, 0);
color: rgb(255, 255, 255);
font-size: 12pt;
font-family: &quot;Segoe UI Light&quot;;
margin-bottom: 10px;</string>
         </property>
         <property name="text">
          <string>Eingabe der Versuchsparameter</string>
         </property>
         <property name="indent">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="styleSheet">
          <string notr="true">QScrollArea {
background-color: rgba(0, 0, 0, 0);
}

QLineEdit {
height: 20px;
font-size: 12pt;
font-family: &quot;Segoe UI Light&quot;;
padding-left: 10px;
padding-bottom: 5px;
border-bottom: 2px solid rgb(255, 255, 255);
border-radius: 0px;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);
}

QLineEdit:hover {
border-bottom: 2px solid rgb(0, 255, 213);
border-radius: 0px;
}

QLineEdit:focus {
border-bottom: 2px solid rgb(0, 255, 213);
border-radius: 0px;
}

QLabel {
background-color: rgba(0, 0, 0, 0);
color: rgb(255, 255, 255);
font-size: 8pt;
font-family: &quot;Segoe UI Light&quot;;
font-weight: bold;
}

QScrollBar:vertical {
background-color: rgba(0, 0, 0, 0);
width: 15px;
margin: 15px 3px 15px 3px;
border: 1px transparent #2A2929;
border-radius: 4px;
}

QScrollBar::handle:vertical {
background-color: #fff;
min-height: 5px;
border-radius: 4px;
}

QScrollBar::handle:vertical:hover {
background-color: rgb(0, 255, 213);
}

QScrollBar::sub-line:vertical {
margin: 3px 0px 3px 0px;
border-image: url(:/);
height: 10px;
width: 10px;
subcontrol-position: top;
subcontrol-origin: margin;
}

QScrollBar::add-line:vertical {
margin: 3px 0px 3px 0px;
border-image: url(:/);
height: 10px;
width: 10px;
subcontrol-position: bottom;
subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical:hover,QScrollBar::sub-line:vertical:on {
border-image: url(:/);
height: 10px;
width: 10px;
subcontrol-position: top;
subcontrol-origin: margin;
}

QScrollBar::add-line:vertical:hover, QScrollBar::add-line:vertical:on {
border-image: url(:/);
height: 10px;
width: 10px;
subcontrol-position: bottom;
subcontrol-origin: margin;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
background: none;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
background: none;
}</string>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>375</width>
            <height>572</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Autor</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_author">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>Autor</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>* Werkstoff</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame">
             <property name="styleSheet">
              <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QComboBox" name="cb_material">
                <property name="styleSheet">
                 <string notr="true">QComboBox {
height: 20px;
font-size: 12pt;
font-family: &quot;Segoe UI Light&quot;;
padding-left: 10px;
padding-bottom: 5px;
border-bottom: 2px solid rgb(255, 255, 255);
border-radius: 0px;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);
}

QComboBox::drop-down {
background-color: rgba(0, 0, 0, 0);
}

QComboBox::down-arrow {
background-color: rgba(0, 0, 0, 0);
}

QComboBox:hover {
border-bottom: 2px solid rgb(0, 255, 213);
border-radius: 0px;
}

QListView { 
border: 0px; 
border-top-left-radius: 0px;
border-top-right-radius: 0px;
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;
color: rgb(62, 101, 190);
background-color: rgb(255, 255, 255);
font-weight: normal;
selection-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(62, 101, 190, 255), stop:1 rgba(80, 50, 156, 255));
show-decoration-selected: 1;
padding: 8px;
}</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_4">
                <property name="minimumSize">
                 <size>
                  <width>107</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>107</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton {
height: 20px;
font-size: 9pt;
font-family: &quot;Segoe UI Light&quot;;
padding-bottom: 2.5px;
border: 2px solid rgb(255, 255, 255);
border-radius: 12.5px;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);
}

QPushButton:hover {
color: rgb(0, 255, 213);
border: 2px solid rgb(0, 255, 213);
}

QPushButton:pressed {
color: rgb(0, 255, 213);
border: 2px solid rgb(0, 255, 213);
background-color: rgba(0, 255, 213, 0.5);
}
</string>
                </property>
                <property name="text">
                 <string>Bearbeiten</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Feuchtigkeit [%]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_moisture_content">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="placeholderText">
              <string>Feuchtigkeit [%]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>* Schnittrichtung</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cb_cutting_direction">
             <property name="styleSheet">
              <string notr="true">QComboBox {
height: 20px;
font-size: 12pt;
font-family: &quot;Segoe UI Light&quot;;
padding-left: 10px;
padding-bottom: 5px;
border-bottom: 2px solid rgb(255, 255, 255);
border-radius: 0px;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);
}

QComboBox::drop-down {
background-color: rgba(0, 0, 0, 0);
}

QComboBox::down-arrow {
background-color: rgba(0, 0, 0, 0);
}

QComboBox:hover {
border-bottom: 2px solid rgb(0, 255, 213);
border-radius: 0px;
}

QListView { 
border: 0px; 
border-top-left-radius: 0px;
border-top-right-radius: 0px;
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;
color: rgb(62, 101, 190);
background-color: rgb(255, 255, 255);
font-weight: normal;
selection-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(62, 101, 190, 255), stop:1 rgba(80, 50, 156, 255));
show-decoration-selected: 1;
padding: 8px;
}</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>* Drehzahl [U/min]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_rotational_speed">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="cursorPosition">
              <number>0</number>
             </property>
             <property name="placeholderText">
              <string>Drehzahl [U/min]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>* Vorschubgeschwindigkeit [m/min]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_feed_speed">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="placeholderText">
              <string>Vorschubgeschwindigkeit [m/min]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>* SB - Werkstück [mm]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_cutting_width">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="placeholderText">
              <string>Schnittbreite [mm]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>* Schnitttiefe [mm]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_cutting_depth">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="placeholderText">
              <string>Schnitttiefe [mm]</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>* ID-Nummer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cb_tool_id">
             <property name="styleSheet">
              <string notr="true">QComboBox {
height: 20px;
font-size: 12pt;
font-family: &quot;Segoe UI Light&quot;;
padding-left: 10px;
padding-bottom: 5px;
border-bottom: 2px solid rgb(255, 255, 255);
border-radius: 0px;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);
}

QComboBox::drop-down {
background-color: rgba(0, 0, 0, 0);
}

QComboBox::down-arrow {
background-color: rgba(0, 0, 0, 0);
}

QComboBox:hover {
border-bottom: 2px solid rgb(0, 255, 213);
border-radius: 0px;
}

QListView { 
border: 0px; 
border-top-left-radius: 0px;
border-top-right-radius: 0px;
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;
color: rgb(62, 101, 190);
background-color: rgb(255, 255, 255);
font-weight: normal;
selection-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(62, 101, 190, 255), stop:1 rgba(80, 50, 156, 255));
show-decoration-selected: 1;
padding: 8px;
}</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Kommentare</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="le_comments">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="placeholderText">
              <string>Kommentare</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
color: #ffffff;
background-color: rgb(255, 170, 0);
height: 30px;
padding-bottom: 5px;
border-radius: 17px;
font-size: 12pt;
font-family: &quot;Segoe UI Light&quot;;
}

QPushButton:hover {
border: 2px solid rgb(255, 255, 255)
}

QPushButton:pressed {
color: rgba(255, 255, 255, 0.5);
background-color: rgba(255, 170, 0, 0.5);
}</string>
            </property>
            <property name="text">
             <string>Schließen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
color: #ffffff;
background-color: rgb(255, 0, 0);
height: 30px;
padding-bottom: 5px;
border-radius: 17px;
font-size: 12pt;
font-family: &quot;Segoe UI Light&quot;;
}

QPushButton:hover {
border: 2px solid rgb(255, 255, 255)
}

QPushButton:pressed {
color: rgba(255, 255, 255, 0.5);
background-color: rgba(255, 0, 0, 0.5);
}</string>
            </property>
            <property name="text">
             <string>Löschen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
color: #ffffff;
background-color: rgb(0, 221, 51);
height: 30px;
padding-bottom: 5px;
border-radius: 17px;;
font-size: 12pt;
font-family: &quot;Segoe UI Light&quot;;
}

QPushButton:hover {
border: 2px solid rgb(255, 255, 255)
}

QPushButton:pressed {
color: rgba(255, 255, 255, 0.5);
background-color: rgba(0, 221, 51, 0.5);
}</string>
            </property>
            <property name="text">
             <string>Bestätigen</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
